.eslintrc.js

```diff
@@ -45,6 +42,10 @@ module.exports = {
             'error',
             'always'
         ],
+        "prettier/prettier": ["error", {
+           "endOfLine":"auto"
+         }],
+         'no-console': 'off',
```

Comment:
I think we might be able to get rid of this line. I'm not seeing any consoles that would require turning it off.